app/Services/LiquidationService.php

```
@@ -0,0 +1,19 @@
+use App\Models\Stakeholders\Project;
+
+class LiquidationService
+{
+    public static function checkAllocation($project_id)
+    {
+        $total = PaymentRequestDetails::where('project_id', $project_id)
+            ->where('type', PaymentRequestType::LIQUIDATION->value)
+            ->sum('total');
+        $projectAllocation = Project::find($project_id)->allocation;
+        return $total > $projectAllocation;
+    }
```

PHPMD (2.15.0)

The parameter $project_id is not named in camelCase. (undefined)

(CamelCaseParameterName)